src/components/cards.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, CSSProperties } from 'react';
+import { items } from '../data/games';
+import { IGames } from 'interfaces/interfaces';
+import './cards.css';
+
+class Cards extends Component {
+  render() {
+    let i = 0;
+    return (
+      <div className="cards__container">
+        {items.map((el: IGames) => {
+          const backroundImage: CSSProperties = {
+            backgroundImage: `url('../../posters/${i}.jpg')`,
+          };
+          i++;
```

Comment:
Kinda awkward use increment like this, you could take index or even better 'id', it would prevent for using wrong picture here

Reply:
Hello, thanks fot the comment! I agree its better to use id, so i fixed it